### PR TITLE
New module standup with fixes for the issues reported by sanity checks

### PR DESCRIPTION
**Pull request by thomastk**
##### SUMMARY

Fixes for the issues reported by sanity checks

##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

standup
##### ANSIBLE VERSION

```
ansible 2.3.2.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
  python version = 2.7.5 (default, Aug  4 2017, 00:39:18) [GCC 4.8.5 20150623 (Red Hat 4.8.5-16)]
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by thomastk** [comment]
Any idea why I get the following error? 

```
Command "test/sanity/code-smell/boilerplate.sh" returned exit status 2.
>>> Standard Output
== Missing __metaclass__ = type ==
./lib/ansible/modules/monitoring/standup.py

== Missing from __future__ import (absolute_import, division, print_function) ==
./lib/ansible/modules/monitoring/standup.py
```
Please help to fix it.

**Comment by thomastk** [comment]
Please let me know what is the next step in the module submission process as it seems all the CI tests pass now. I didn't know that the fixes checked into the same PR would get CI tested automatically, and, ended up closing the first PR 31402.

File: lib/ansible/modules/monitoring/standup.py [hunk 1]
```diff
@@ -0,0 +1,617 @@
...
+    
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+author: "Thomas Kurian Theakanath (thomastk@)"
```
Comment (gundalow) [inline lib/ansible/modules/monitoring/standup.py]: `@thomastk`

File: lib/ansible/modules/monitoring/standup.py [hunk 2]
```diff
@@ -0,0 +1,617 @@
...
+  output_compare:
+     type: number
+     value: 3
+     operator: EQ
+'''
+
+RETURN = '''
+checks:
+    description: The checks with results of running check and heal commands.
+    type: dict
```
Comment (gundalow) [inline lib/ansible/modules/monitoring/standup.py]: `type: complex`

Have a look at other modules with the above line for how to format this

**Comment by bcoca** [comment]
Correct me if i'm wrong, this mostly seems like a very specialized version of this:

```yaml
- block:
   - action: get_some_facts
     register: results

   - assert:
        that:
            - result.stuff == 'stuff i expect'
  rescue:
      - command: heal
```

File: lib/ansible/modules/monitoring/standup.py (hunk 1 above)
Comment (thomastk) [reply lib/ansible/modules/monitoring/standup.py]: Fixed. Thanks!

**Comment by thomastk** [comment]
@bcoca You are right, essentially the module also runs the commands, evaluates the output and runs heal commands if needed as it could be done using the block construct. However, standup module  would be more helpful to the users of Ansible in the following ways:
- The checks are externalized in a YAML file. To implement a new test/monitoring suite, the QA or Infra engineers can develop the related YAML file with checks that playbook developers or the QA/Infra engineers themselves can integrate easily. 
-  It is specifically built for verifying and monitoring a group of hosts using multiple checks. The RETURN value is constructed for piping into other monitoring modules to publish the status of a cluster via downstream monitoring tools like Datadog, Nagios or Zabbix, besides using the tool for one-off verification of infra clusters.
- Tagging infra with application roles is very common for the types of checks thatcould be run on them for verification. standup module provides option for role based execution of checks that would help to organize the checks defined in test/monitoring suits better.

File: lib/ansible/modules/monitoring/standup.py (hunk 2 above)
Comment (thomastk) [reply lib/ansible/modules/monitoring/standup.py]: Checked in the reformatted RETURN block.

**Comment by thomastk** [comment]
I am going through this process first time. Please let me know if any details needed from my side.

**Pull request closed without merging**